### PR TITLE
feat(hosting.ftp): add sftp information in user table (pro offer)

### PR DESCRIPTION
**Pull request by lucas-chaigne**
ref: DTRSD-43496

Signed-off-by: Lucas Chaigne <lucas.chaigne@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/webhosting-sftp-doc`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #DTRSD-43496
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

In webhosting manager, ftp-ssh tab, for pro and perf offers : user table is updated to add sftp information

## Related

- #5988
- #5989
- #5992
- #5993

### CODE REVIEW

File: packages/manager/apps/web/client/app/hosting/ftp/FTP.html [hunk 1]
```diff
@@ -470,19 +470,19 @@
                                 <tr>
                                     <th
                                         scope="col"
-                                        data-translate="hosting_tab_FTP_multiftp_table_header_ftp_login"
```
Comment (antleblanc) [inline packages/manager/apps/web/client/app/hosting/ftp/FTP.html:473]: **question:** Does following translation keys could be removed from translation file then? (only for `Messages_fr_FR.json`)

- hosting_tab_FTP_multiftp_table_header_ftp_login
- hosting_tab_FTP_multiftp_table_header_ssh_login
- hosting_tab_FTP_multiftp_table_header_ftp_state

File: packages/manager/apps/web/client/app/hosting/ftp/hosting-ftp-constants.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+export const SSH_STATE = {
+  ENABLED: 'active',
+  SFTP_ONLY: 'sftponly',
+  DISABLED: 'none',
```
Comment (JeremyDec) [inline packages/manager/apps/web/client/app/hosting/ftp/hosting-ftp-constants.js:4]: Why not naming the constants to match their value?

File: packages/manager/apps/web/client/app/hosting/ftp/FTP.html (hunk 1 above)
Comment (lucas-chaigne) [reply packages/manager/apps/web/client/app/hosting/ftp/FTP.html:473]: Hello,

thanks for the review hosting_tab_FTP_multiftp_table_header_ftp_login and hosting_tab_FTP_multiftp_table_header_ssh_login can be removed.
hosting_tab_FTP_multiftp_table_header_ftp_state is still used.

File: packages/manager/apps/web/client/app/hosting/ftp/hosting-ftp-constants.js (hunk 2 above)
Comment (lucas-chaigne) [reply packages/manager/apps/web/client/app/hosting/ftp/hosting-ftp-constants.js:4]: The idea was to match text displayed to the user, but thinking about it it makes more sense to name it as you propose.

**Review by antleblanc — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Pull request merged**